src/viur/core/skeleton.py

```diff
@@ -409,6 +409,9 @@ def setBoneValue(cls, skelValues: t.Any, boneName: str, value: t.Any,
         if not isinstance(bone, BaseBone):
             raise ValueError(f"{boneName} is no valid bone on this skeleton ({skelValues})")
         skelValues[boneName]  # FIXME, ensure this bone is unserialized first
+        if value is None:
+            skelValues[boneName] = None
+            return True
```

Comment:
Doesn't deserialization make sense when not being in the `None`-case? 
```suggestion
        if value is None:
            skelValues[boneName] = None
            return True

        _ = skelValues[boneName]  # ensure the bone is being unserialized first
```